src/ResourceManager/DevSpaces/Commands.DevSpaces/help/Get-AzureRmDevSpacesController.md

```diff
@@ -0,0 +1,165 @@
+---
+external help file: Microsoft.Azure.Commands.DevSpaces.dll-Help.xml
+Module Name: AzureRM.DevSpaces
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.devspaces/get-azureevspacescontroller
+schema: 2.0.0
+---
+
+# Get-AzureRmDevSpacesController
+
+## SYNOPSIS
+Get or list Azure DevSpaces controller.
+
+## SYNTAX
+
+### ListDevSpacesControllerParameterSet (Default)
+```
+Get-AzureRmDevSpacesController [[-ResourceGroupName] <String>] [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### DevSpacesControllerNameParameterSet
+```
+Get-AzureRmDevSpacesController [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
```

Comment:
regen once you make changes to ensure the help reflects those changes.

Reply:
Done